### PR TITLE
[WIP]bma180 support added for accelerometer

### PR DESCRIPTION
**Pull request by ho-dor**
Fixes First steps for issues #1840 

**Changes**: bma180 support for accelerometer

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: WIP

### CODE REVIEW

File: app/src/main/java/io/pslab/sensors/SensorBMA180.java [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+import io.pslab.communication.peripherals.I2C;
+import io.pslab.communication.sensors.BMA180;
+import io.pslab.others.ScienceLabCommon;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Harsh on 6/6/18.
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/sensors/SensorBMA180.java]: Really?

**Comment by CloudyPadmal** [comment]
@ho-dor did you check if this works?

**Comment by ho-dor** [comment]
I am still working on this hence the WIP tag.

**Comment by CloudyPadmal** [comment]
@ho-dor what's the progress of this? It seems like a big copy pasted code with no progress. Even Harsh's name is there.

**Comment by ho-dor** [comment]
Well, the working is similar to mpu6050. It is dependent on the i2c working so once the i2c reading issue gets resolved , this can progress.

**Comment by CloudyPadmal** [comment]
Open when ready

**Pull request closed without merging**